server/controllers/Contacts.js

```diff
@@ -97,6 +103,7 @@ module.exports.deleteContact = async (req, res) => {
 
     // Validate request
     if (!userId || !contactId) {
```

Comment:
userId check can probably be removed because the route cannot be accessed without it.